tests/snmpsim/generex-ups-card.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.1.0|4|CS121
```

Comment:
That's still fine, we just should have full data to test against where possible.